rockset/provider_test.go

```diff
@@ -90,6 +92,24 @@ func getHCL(filename string) string {
 	return hcl
 }
 
+// getHCLTemplate returns a rendered HCL config
+func getHCLTemplate(filename string, data any) string {
+	hclPath := filepath.Join("..", "testdata", filename)
```

Comment:
Not a must, moreso asking out loud, but would we want to do a bit of DRY by calling `getHCL(filename)` for the first part of the function? It's only a few lines, so maybe not worth it.